site/app/templates/admin/Configuration.twig

```diff
@@ -148,10 +148,30 @@
                 </div>
             </div>
         </div>
-        <div class="post-panel-btn">
-            <button class="btn btn-primary" style="float: right" type="submit" form="configForm">
-                <i class="fa fa-save fa-fw"></i> Submit
-            </button>
-        </div>
     </form>
-</div>
\ No newline at end of file
+    <script>
+        $(function() {
+            $("input,textarea").on("change", function() {
+                let formData = new FormData();
+                formData.append('csrf_token', csrfToken);
+                let entry;
+                if(this.type === "checkbox") {
+                    entry = $(this).is(":checked");
+                }
+                else {
+                    entry = this.value;
+                }
+                formData.append("name", this.name);
+                formData.append("entry", entry);
+
+                $.ajax({
+                    url: "{{ core.buildUrl({'component': 'admin', 'page': 'configuration', 'action': 'update'})|raw }}",
+                    data: formData,
+                    type: "POST",
+                    processData: false,
+                    contentType: false
+                });
+            });
+        });
+    </script>
+</div>
```

Comment:
Make a .js file for the ajax call.  Also, look into using `$.getJSON` so you can interpret your json response on the client and tell the user if something went wrong.